frontend/src/api/ClientsRepository.js

```diff
@@ -10,6 +10,11 @@ export default {
     getRoles(clientId){
         return kcRequest().then(axiosInstance => axiosInstance.get(`${resource}/${clientId}/roles`));
     },
+    
+    // REVIEW - Do we need this API in UMS, or should it use the KC API?
```

Comment:
We actually want to entirely remove the UMC's dependency on the KC API. Our reasons for doing so are documented in [backend/README.md](https://github.com/bcgov/moh-keycloak-user-management/blob/master/backend/README.md).